docs/source/release/v3.14.0/indirect_inelastic.rst

```diff
@@ -37,6 +37,8 @@ Bugfixes
 
 - The parameter values for a selected spectrum are now updated properly when a Fit is run using the Fit String 
   option in ConvFit.
+- An unexpected crash is prevented when Plot Current Preview is clicked when no data is loaded. A meaningful error
```

Comment:
Do we know if this was in 13.3 release? If it wasn't then I would say that it might not belong in the release notes as it is something that we would have introduced during development and user (unless they are using the nightly) would be unaware of it. 
Although if we don't know if it was in 13.3 or not, the safe route is to just add it too the release notes!